### DISCUSSION TITLE
Can I create a form for a table?

### DISCUSSION THREAD

**Discussion by lakridserne** [nocodb/nocodb#548] [Q&A] [answered]
I saw that it was possible to create a form view in the documentation, but not in the product itself.

I would kinda like to be able to create forms like Google forms, where when the user fills in the fields, it'll fill in a row in the table. Is that possible?

**Comment by o1lab** [comment, marked as answer]
We 're working on form - will be out soon. Tracked in [roadmap](https://github.com/nocodb/nocodb/projects/1).
